### PR TITLE
Feat/encepp update

### PR DESCRIPTION
**Pull request by BrendaHijmans**
-Bump version to 3.4
-Add EMA to profiles for all EMA data items
- ....

### CODE REVIEW

File: data/datacatalogue3/CatalogueOntologies/Areas of information ds.csv [hunk 1]
```diff
@@ -44,8 +46,7 @@ Diagnoses codes included: CCS,CCS,,Diagnoses codes included
 Diagnoses codes included: EDC,EDC,,Diagnoses codes included
 Diagnoses codes included: Not coded (Free text),Not coded (Free text),,Diagnoses codes included
 Diagnoses codes included: Other,Other,,Diagnoses codes included
-Hospital admission,,Is information on hospital admission available?,Clinical data
-Hospital discharge,,Is information on hospital discharge available?,Clinical data
+Hospital admission discharge,,Is information on hospital admission and discharge available?,Clinical data
```
Comment (mswertz) [inline data/datacatalogue3/CatalogueOntologies/Areas of information ds.csv:49]: Why merge these?

File: data/datacatalogue3/CatalogueOntologies/Organisation roles.csv [hunk 2]
```diff
@@ -1,5 +1,7 @@
 name
 Data originator
+Data provider
```
Comment (mswertz) [inline data/datacatalogue3/CatalogueOntologies/Organisation roles.csv:3]: can we please start adding definitions? what is difference between originator, holder and provider. I think I know but I am not sure ;-)

File: data/scripts/molgenis-model-update/README [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Use virtual env to get a consistent python environment.
+
+1. Clone the github repository
+
+    'git clone git@github.com:molgenis/molgenis-emx2'
+
+2. Create a virtual python environment at the location of the script
+
+    'cd data\scripts\molgenis-model-update'
```
Comment (mswertz) [inline data/scripts/molgenis-model-update/README:20]: slashes wrong way?

**Review by mswertz — COMMENTED** [review]
build still failing.

**Comment by BrendaHijmans** [comment]
this was not ready for review. I am making a new PR.

**Pull request closed without merging**